Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM ruby:2.5.5
+FROM ruby:2.6.5
+
+ENV DB_PASSWORD openregister
+ENV DB_USER openregister-info
+ENV DB_DB openregister-info_development
+ENV DB_HOST db
```

Comment:
Maybe I am missing something, but why not move this to docker-compose.yml too?

Reply:
I think I was looking at the [admin Dockerfile](https://github.com/alphagov/govwifi-admin/blob/master/Dockerfile) which encodes them in there but I think you're right it makes more sense to have them in the Compose file. Have pushed a commit.